WordPress/src/test/java/org/wordpress/android/ui/uploads/LocalDraftUploadStarterConcurrentTest.kt

```diff
@@ -0,0 +1,83 @@
+package org.wordpress.android.ui.uploads
+
+import android.arch.core.executor.testing.InstantTaskExecutorRule
+import com.nhaarman.mockitokotlin2.any
+import com.nhaarman.mockitokotlin2.doReturn
+import com.nhaarman.mockitokotlin2.eq
+import com.nhaarman.mockitokotlin2.mock
+import com.nhaarman.mockitokotlin2.times
+import com.nhaarman.mockitokotlin2.verify
+import kotlinx.coroutines.Dispatchers
+import kotlinx.coroutines.runBlocking
+import org.junit.Rule
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.mockito.junit.MockitoJUnitRunner
+import org.wordpress.android.fluxc.model.PostModel
+import org.wordpress.android.fluxc.model.SiteModel
+import org.wordpress.android.fluxc.store.PostStore
+import org.wordpress.android.util.NetworkUtilsWrapper
+
+/**
+ * Tests for structured concurrency in [LocalDraftUploadStarter].
```

Comment:
I'd suggest adding a comment with explanation this test suite should be separated from the `LocalDraftUploadStarterTest` as it contains non-deterministic tests.

Reply:
Done in 3e675a790c3a87219dd1f454e5e9af2d768ef517.